docs/guide/data_analysis.ipynb

```diff
@@ -127,7 +127,20 @@
       "source": [
         "## Load data\n",
         "\n",
-        "We can use utilities in ReCirq to query the filesystem and load in a dataset. Please recall that all tasks have an associated `EXPERIMENT_NAME` and a `dataset_id` which define the top two hierarchies in the filesystem. We import these values from the data collection script to ensure consistency."
+        "We can use utilities in ReCirq to query the filesystem and load in a dataset. Please recall that all tasks have an associated `EXPERIMENT_NAME` and a `dataset_id` which define the top two hierarchies in the filesystem. We import these values from the data collection script to ensure consistency.\n",
+        "\n",
+        "If you're running this notebook in Colab or you haven't yet gone through the Data Collection tutorial, we will download a pre-generated copy of the data for analysis."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "75m62Mlkphm2"
+      },
+      "outputs": [],
+      "source": [
+        "recirq.fetch_guide_data_collection_data()"
```

Comment:
Got this error in Colab:

```
AttributeError: module 'recirq' has no attribute 'fetch_guide_data_collection_data'
```

Reply:
That function is added in this PR, so you can't really test it until this is merged

Reply:
Gotcha. LGTM and we can test once it lands. Thanks